src/schema/v1/artwork_or_edition_set.ts

```diff
@@ -0,0 +1,8 @@
+import { GraphQLUnionType } from "graphql"
+import { ArtworkType } from "./artwork"
+import { EditionSetType } from "./edition_set"
+
+export const ArtworkOrEditionSetType = new GraphQLUnionType({
```

Comment:
I ran into issues having a single type that was consumed by the v1 and v2 schemas separately ([this issue](https://github.com/MichalLytek/type-graphql/issues/396), in particular). Curious if @zephraph ran into this as well with the original v1/v2 work?

The solution to have the type copied over seems OK.

Although generally, excited to 🔪 v1 😄 whenever that is possible.

Reply:
Hmm, why add these types (and make the schema updates) to V1? It might be cleaner to only stitch in the new field which refers to this new type, in V2 (vs. stitching it in to V1 and V2).

Reply:
This is a good point. I didn't look into how to only stitch in fields for a certain schema (v1 or v2), but I will!

Reply:
One example of that is in [gravity stitching](https://github.com/artsy/metaphysics/blob/d6f3dc7636d2d2f73d62e350b6e27d0b49824851/src/lib/stitching/gravity/stitching.ts#L110).